docs/documentation/unity/samples/demo.md

```diff
@@ -0,0 +1,34 @@
+## Initialize and connect to a demo conference
+This example demonstrates how to initialize the Unity plugin and connect to a demo conference using visual scripting. The Demo conference node allows connecting to a conference that has several bots injecting audio. This makes it easier for testing the connection during the prototyping phase.
+
+To start fresh, you can create a new project from Unity Hub using one of the provided templates, such as `Core 3D`. The following video shows the workflow.
+
+<div style="position: relative; padding-bottom: 56.25%; height: 0;"><iframe src="https://www.loom.com/embed/2b41901f211a409c96dcac2a178fabcf" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 80%; height: 100%;"></iframe></div>
```

Comment:
Sorry for this, but we need to changeb a bit the video with the new Initialize and GetToken node